versions/z-/zoe.json

```
@@ -0,0 +1,24 @@
+{
+  "versions": [
+    {
+      "git-tree": "24d50f143b93df2ab26fc6420ef6648cac18005a",
+      "version": "3.0",
+      "port-version": 1
+    },
+    {
+      "git-tree": "b1c5c5bb8d55ed4a3b7a763dc19c69666694e6c4",
+      "version": "3.2",
+      "port-version": 0
+    },
+    {
+      "git-tree": "bbac35ef9a8843e0f75c2c8823c386909686fa50",
+      "version": "3.1",
+      "port-version": 0
+    },
```

```suggestion
  "versions": [
    {
      "git-tree": "24d50f143b93df2ab26fc6420ef6648cac18005a",
      "version": "3.0",
      "port-version": 1
    },
    {
      "git-tree": "b1c5c5bb8d55ed4a3b7a763dc19c69666694e6c4",
      "version": "3.2",
      "port-version": 0
    },
    {
      "git-tree": "bbac35ef9a8843e0f75c2c8823c386909686fa50",
      "version": "3.1",
      "port-version": 0
    },
```
Just run command `./vcpkg x-add-version zoe --overwrite-version`